### DISCUSSION TITLE
Redis helm strange behaviour

### DISCUSSION THREAD

**Discussion by Kapsztajn** [argoproj/argo-cd#8104] [Q&A]
Hello,
Im currently trying to implement ArgoCD in K8 cluster with apps in Helm. A part of needed services is redis. Im using it in standalone mode from Bitnami helm chart. I have this strange bug? in which after every commit I make (even if it doesnt change anything in redis values/helm) it keeps changing master pod and statefulset for redis. It creates new controllerrevision and create new pod for unknown reasons for me.
Did someone occured this in some others app? Did I miss something during configuration?

**Comment by joaofhenriques** [comment]
Having the same issue. Did you find any solution ?
